src/Containers/LeftSideBar/CollapsibleHeader.tsx

```diff
@@ -39,9 +40,10 @@ export const CollapsibleHeader: React.FC<CollapsibleHeaderProps> = ({
     icon,
     category,
     isCollapsedProp,
+    position,
     ...props
 }) => {
-    const [isCollapsed, setIsCollapsed] = useState(isCollapsedProp);
+    let [isCollapsedArr, setIsCollapsedArr] = isCollapsedProp;
```

Comment:
put this back as useState and have a useEffect or remove this entirely and just let the header be controlled by the props instead of state